lib/modules/datasource/sbt-plugin/index.ts

```diff
@@ -9,10 +9,11 @@ import { SbtPackageDatasource } from '../sbt-package';
 import { getLatestVersion, parseIndexDir } from '../sbt-package/util';
 import type { GetReleasesConfig, ReleaseResult } from '../types';
 
+export const SBT_MVN_REPO = 'https://repo1.maven.org/maven2';
```

Comment:
The `MAVEN_REPO` constant from the `maven` datasource should be used instead

Reply:
good catch

Reply:
I've merged @zharinov 's pull request to address this.